src/handlers/handle_config.py

```diff
@@ -72,7 +72,7 @@ def createConfig():
     config['SFTP - Remote Server']['Server'] = '0.0.0.0'
     config['SFTP - Remote Server']['Username'] = 'user'
     config['SFTP - Remote Server']['Password'] = 'password'
-    config['SFTP - Remote Server'][';'] = 'If a different Port for SFTP needs to be used (Works only for SFTP)'
+    config['SFTP - Remote Server'][';'] = 'If a different Port for SFTP needs to be used'
     config['SFTP - Remote Server']['SFTPPort'] = '22'
```

Comment:
Idea: 
Create a new config option under here for FTP default port configuration which is 21 as default